app/code/Magento/Catalog/Api/CategoryListDeleteBySkuInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Catalog\Api;
+
+/**
+ * @api
+ * @since 100.0.2
+ */
+interface CategoryListDeleteBySkuInterface
+{
+    /**
+     * Delete by skus list
+     *
+     * @param int   $categoryId
+     * @param array $productSkuList
+     * @return bool
+     *
+     * @throws \Magento\Framework\Exception\CouldNotSaveException
+     * @throws \Magento\Framework\Exception\InputException
+     */
+    public function deleteBySkus($categoryId, array $productSkuList);
```

Comment:
Please add type to $categoryId param and return function type

Reply:
@sarron93

Reply:
@VladimirZaets, ready